duckduckgo/duckduckgo-autofill
# Send a breakage report flag when autofill is "activated" on the page

**Reviewer:** @GioSensation 
**Asana:** https://app.asana.com/0/1163321984198618/1205204138932467/f

## Description
This sends an "activation" flag to the background script of the extension.
- This flag will be attached to the pixel request, if the user reports breakage
- in practice, it seems like the flag will be sent all the time, unless autofill is disabled in remote config. This is because autofill is making potentially breaking changes even if autofill is not enabled in extension settings.
- other platforms don't support the breakage flags yet, see https://app.asana.com/0/72649045549333/1203966337678070/f for more details

## Steps to test
- build the extension
- navigate to any page where autofill is not disabled through the remote config. (it doesn't need to be enabled in settings)
- open the devtools for the background page (or service worker for mv3)
- send a breakage report
- observe `debugFlags=autofill` in the report pixel parameters


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205204236992044
